src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsParser.java

```diff
@@ -44,4 +45,8 @@ public ProjectsParser(Gson gson) {
     public ProjectInfo parseSingleProjectInfo(JsonElement result) {
         return gson.fromJson(result, ProjectInfo.class);
     }
+
+    public String generateProjectInput(ProjectInput input) {
```

Comment:
Yes, you are absolutely right. I was just unsure about it because it's not done this way in other classes and also the name `Parser` is a bit misleading. But it's probably a good idea to think about a renaming and move all the other usages of Gson in *Client into the *Parsers.

So I think we will leave it as is in this change and think about it in the future. Any suggestion for a better naming?